src/run/lib/Tree_sitter_error.atd

```diff
@@ -0,0 +1,28 @@
+(*
+   Type definition for json errors exported by the test parsers.
+*)
+
+type node_kind <ocaml from="Tree_sitter_bindings.Tree_sitter_output"
+                      t="node_kind"> = abstract
+
+type position <ocaml from="Tree_sitter_bindings.Tree_sitter_output"
+                     t="position"> = abstract
+
+type error_kind = [
+  | Internal (* a bug *)
+  | External (* malformed input or bug, but we don't know *)
+] <ocaml repr="classic">
+
+(*
+   Simplified type suitable for json export.
+   The original type is Tree_sitter_error.t (in Tree_sitter_error.mli).
+*)
+type json_error = {
+  kind: error_kind;
+  msg: string;
+  file: string;
+  start_pos: position;
+  end_pos: position;
+  substring: string;
+  ?error_class: string option;
```

Comment:
maybe expose also the parent, sibling, first_child as subfields, so a python script does not have to reparse the error_class string?

Reply:
This is meant as an identifier of a class of errors. I expect it might change, and I prefer to deal with it in ocaml than in python.